woocommerce-wirecard-checkout-page/classes/class-woocommerce-wcp-config.php

```diff
@@ -156,4 +156,15 @@ public function get_secret() {
 		$config = $this->get_client_config();
 		return $config['SECRET'];
 	}
+
+    /**
+     * Checks is ratepay enabled
+     *
+     * @since 1.3.8
+     * @param $payment_method
+     * @return boolean
+     */
+    public function is_ratepay_enabled($payment_method) {
+        return $this->_settings[$payment_method] == 'ratepay';
+    }
```

Comment:
you could write `return $this->_settings[$payment_method . '_provider'] == 'ratepay';` to keep the calls cleaner